### PR TITLE
Add jupyterlab-classic

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jtpio** [comment]
Looks like the windows build is failing, but not sure why.

**Comment by bollwyvl** [comment]
Looks like [unicode issues](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=255703&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=649):

```
    Running command 'C:\Miniconda\conda-bld\jupyterlab-classic_1608826272346\_h_env\python.exe' 'C:\Miniconda\conda-bld\jupyterlab-classic_1608826272346\_h_env\lib\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\VSSADM~1\AppData\Local\Temp\tmpgi06nd7l'
    Traceback (most recent call last):
      File "C:\Miniconda\conda-bld\jupyterlab-classic_1608826272346\_h_env\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 280, in <module>
        main()
      File "C:\Miniconda\conda-bld\jupyterlab-classic_1608826272346\_h_env\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 263, in main
        json_out['return_val'] = hook(**hook_input['kwargs'])
      File "C:\Miniconda\conda-bld\jupyterlab-classic_1608826272346\_h_env\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 133, in prepare_metadata_for_build_wheel
        return hook(metadata_directory, config_settings)
      File "C:\Miniconda\conda-bld\jupyterlab-classic_1608826272346\_h_env\lib\site-packages\setuptools\build_meta.py", line 174, in prepare_metadata_for_build_wheel
        self.run_setup()
      File "C:\Miniconda\conda-bld\jupyterlab-classic_1608826272346\_h_env\lib\site-packages\setuptools\build_meta.py", line 158, in run_setup
        exec(compile(code, __file__, 'exec'), locals())
      File "setup.py", line 74, in <module>
        long_description = fh.read()
      File "C:\Miniconda\conda-bld\jupyterlab-classic_1608826272346\_h_env\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 918: character maps to <undefined>

```

File: recipes/jupyterlab-classic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (bollwyvl) [inline recipes/jupyterlab-classic/meta.yaml:15]: you may wish to try `--no-deps`, if the pip solver is the issue...

File: recipes/jupyterlab-classic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "jupyterlab-classic" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "94e83b6fbfcf0e3ad195883ea8794a7ac2135ca7177d6a9e0ebd60ddc21a175a" %}
+
+package:
+  name: {{ name|lower }}
```
Comment (bollwyvl) [inline recipes/jupyterlab-classic/meta.yaml:6]: already lowercase :P

File: recipes/jupyterlab-classic/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - python >=3.6
+    - jupyterlab >=3.0.0,<4
+
+test:
+  imports:
+    - jupyterlab_classic
+  commands:
+    - pip check
+    - jupyter classic --help
```
Comment (bollwyvl) [inline recipes/jupyterlab-classic/meta.yaml:32]: Perhaps add a `jupyter labextension list` as well, just so there's some record of `data_files` having been moved around properly.

**Comment by bollwyvl** [comment]
Made an upstream PR for fixing the windows thing...

File: recipes/jupyterlab-classic/meta.yaml (hunk 3 above)
Comment (jtpio) [reply recipes/jupyterlab-classic/meta.yaml:32]: Thanks for the suggestion, sounds good :+1:

**Comment by jtpio** [comment]
> Made an upstream PR for fixing the windows thing...

Thanks! I'll publish a new release later today with the fix.

**Comment by jtpio** [comment]
Looks better now with the upstream fix and after adding the entry points.

File: recipes/jupyterlab-classic/meta.yaml [hunk 4]
```diff
@@ -13,6 +13,8 @@ build:
   number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - jupyter-classic = jupyterlab_classic.app:main
```
Comment (bollwyvl) [inline recipes/jupyterlab-classic/meta.yaml:17]: :+1:

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review (unless @jtpio violently disagrees!)

File: recipes/jupyterlab-classic/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/jupyterlab-classic/meta.yaml:15]: no-deps should be default in our builds, no need to specify it.

**Comment by ocefpaf** [comment]
> Made an upstream PR for fixing the windows thing...

In a way, while that can be fixed upstream, it is an annoying bug in conda IMO. A pip install of that would work. I've seen that in a few packages that has non-standard character files on Windows.

File: recipes/jupyterlab-classic/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
+{% set name = "jupyterlab-classic" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "0f918c987ea88d19c2a099cf45e3c4b29ae014e85b335e7abdb2a499280fe46d" %}
```
Comment (ocefpaf) [inline recipes/jupyterlab-classic/meta.yaml:3]: No need to create a variable that will be used in a single  place.
BTW, try [grayskull](https://github.com/conda-incubator/grayskull) to create your recipes. Things like these will be done automatically for you.

File: recipes/jupyterlab-classic/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/jupyterlab-classic/meta.yaml:15]: good to know!

**Comment by bollwyvl** [comment]
> In a way, while that can be fixed upstream, it is an annoying bug in conda IMO. A pip install of that would work. I've seen that in a few packages that has non-standard character files on Windows.

Welp, it's tested upstream now. I basically assume things can _always_ go wrong on windows...

File: recipes/jupyterlab-classic/meta.yaml (hunk 5 above)
Comment (jtpio) [reply recipes/jupyterlab-classic/meta.yaml:3]: Nice thanks, grayskull looks great!

Comment (bollwyvl) [reply recipes/jupyterlab-classic/meta.yaml:3]: my kingdom for `grayskull go` and `grayskull rust`...

**Pull request merged**